recipes/wxpython/bld.bat

```diff
@@ -0,0 +1,14 @@
+:: LOTS more to do here!
```

Comment:
Unclear. What else remains to be done?
Can you clean the comments a little bit.